lib/view_model/dashboard/dashboard_view_model.dart

```diff
@@ -536,6 +537,88 @@ abstract class DashboardViewModelBase with Store {
     return resp;
   }
 
+  @observable
+  late bool backgroundSyncNotificationsEnabled = sharedPreferences.getBool(PreferencesKey.backgroundSyncNotificationsEnabled) ?? false;
+
+  @action
+  Future<void> setBackgroundSyncNotificationsEnabled(bool value) async {
+    if (!value) {
+      backgroundSyncNotificationsEnabled = false;
+      sharedPreferences.setBool(PreferencesKey.backgroundSyncNotificationsEnabled, false);
+      return;
+    }
+    PermissionStatus permissionStatus = await Permission.notification.status;
+    if (permissionStatus == PermissionStatus.denied) {
```

Comment:
Minor
wouldn't this be better?
```suggestion
    if (permissionStatus != PermissionStatus.granted) {
```